### PR TITLE
LPS-193250 Publish all Children Objects in a row during their Root Object publication. Block standalone publication for children Objects

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-193250

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java [hunk 1]
```diff
@@ -2021,6 +1882,128 @@ private ObjectDefinition _addUnmodifiableSystemObjectDefinition(
 				).build()));
 	}
 
+	private void _assertBoundedObjectDefinitions(
```
Comment (mateussgn) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java:1878]: Awesome idea!

I just suggest to separate each assertType on it's own method.
For example:
```
    _assertDataBasetableBoundedObjectDefinitions
    _assertPublishFailureBoundedObjectDefinitions
    _assertResourcesBoundedObjectDefinitions
    _assertStatusBoundedObjectDefinitions
```

**Review by mateussgn — CHANGES_REQUESTED** [review]
Great work, @p-albuquerque!

**Comment by p-albuquerque** [comment]
Resent in: https://github.com/liferay-objects/liferay-portal/pull/2505

**Pull request closed without merging**